presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -684,27 +684,25 @@ std::vector<std::string> PrestoServer::registerConnectors(
       auto catalogName =
           fileName.substr(0, fileName.size() - kPropertiesExtension.size());
 
-      auto connectorConf = util::readConfig(entry.path());
+      auto connectorConfig = ConnectorConfig::instance();
+      connectoConfig->initialize(entry.path());
```

Comment:
We have multiple connectors and each of them has a config.
It seems we don't need a ConnectorConfig's instance to stay, since we just take prop maps from them and give it to the connectors.
So, no need to have ConnectorConfig::instance() method - just create one ConnectorConfig object (not a pointer) on the stack inside the loop.